cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfJointComponent.h

```diff
@@ -22,6 +23,13 @@ class UUrdfJointComponent : public UPhysicsConstraintComponent
 public:
     void initializeComponent(UrdfJointDesc* joint_desc, UUrdfLinkComponent* parent_link, UUrdfLinkComponent* child_link);
 
+    float getAction();
+    void applyAction(float action);
+
+    UUrdfLinkComponent* parent_link_;
```

Comment:
I think these pointers should be private. I know this isn't our usual strategy for pawns, but I think it makes sense for this component.